dev/build_google_image_with_gcloud.sh

```diff
@@ -0,0 +1,380 @@
+#!/bin/bash
+#
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPT_DIR=$(dirname $0)
+ADD_CLOUD_LOGGING=true
+ADD_MONITORING=true
+UPDATE_OS=true
+
+TIME_DECORATOR=$(date +%Y%m%d%H%M%S)
+REPOSITORY_URL=https://dl.bintray.com/spinnaker/debians
+INSTALL_SCRIPT=https://dl.bintray.com/spinnaker/scripts/InstallSpinnaker.sh
+
+BASE_IMAGE=ubuntu-1404-lts
```

Comment:
The idea here is you can pass in either so you can create off a specific image or off an image family. I called it BASE_IMAGE but maybe it should be something else (but not FAMILY)